### PR TITLE
4816 listing organization users

### LINKED ISSUE #4816: Listing organization's users in organization management view
Listing organization users (staff members, teachers, students) in User tab with following details:

* User name
* User email (obfuscated)
* User profile avatar

View should be separated in 2 sections, staff members (that includes admins and teachers) and students.

Both sections content should use 2 columns as we don't need Inactive column at this point. 

Child of #4749

### LINKED ISSUE #4817: Listing organization's courses in organization management view
Listing organization's courses with not yet specified filters. Idea is the same as in course picker view that left aside navigation has filters and right column lists courses.

* Course's name
* Course's curriculumn

ie. **MAA1 - Course's name (OPS 2016)**

Child of #4749

### PR DESCRIPTION
**Pull request by avukkonen**
I accidently also workspaces.

Closes #4816  
Closes #4817

### CODE REVIEW

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/organization/body.tsx [hunk 1]
```diff
@@ -22,4 +22,9 @@ export default class OrganizationManagementBody extends React.Component<Organiza
         </ScreenContainer>
     </div>);
   }
-}
\ No newline at end of file
+}
+
+let kinostaa = true;
```
Comment (anttiviljakainen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/organization/body.tsx]: WUT?

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/organization/body/application/workspaces.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        {this.props.coursepickerCoursesState === "LOADING_MORE" ? <ApplicationListItem className="loader-empty"/> : null}
+        </ApplicationList>
+    );
+  }
+}
+
+function mapStateToProps(state: StateType){
+  return {
+    i18n: state.i18n,
+    coursepickerCoursesState: state.courses.state,
```
Comment (onzag) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/organization/body/application/workspaces.tsx]: We might not want to have states named coursepicker if they are not in coursepicker

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/main-function/users/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    //Create the payload for updating all the communicator properties
+    let payload:GuiderPatchType = {
+      state: "READY",
+      students: (concat ? guider.students.concat(actualStudents) : actualStudents),
+      hasMore
+    }
+    
+    //And there it goes
+    dispatch({
+      type: "UPDATE_GUIDER_ALL_PROPS",
```
Comment (onzag) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/main-function/users/helpers.ts]: users helper function  updates guider?...

These all seem guider related.

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/organization/body/application/workspaces.tsx (hunk 2 above)
Comment (onzag) [reply muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/organization/body/application/workspaces.tsx]: Did we ever decide to separate the standard courses vs the organization courses?

Comment (avukkonen) [reply muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/organization/body/application/workspaces.tsx]: > Did we ever decide to separate the standard courses vs the organization courses?

We did, it's happening in the other branch

**Pull request merged**